mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -475,13 +489,37 @@
     <!-- Default name for a new collection in "name new collection" step of the collection creator. %d is a placeholder for the number of collections-->
     <string name="create_collection_default_name">Collection %d</string>
 
+    <!-- Share -->
+    <!-- Share screen header -->
+    <string name="share_header">Send and Share</string>
+    <!-- Sub-header in the dialog to share a link to another app -->
+    <string name="share_link_subheader">Share a link</string>
+    <!-- Sub-header in the dialog to share a link to another sync device -->
+    <string name="share_device_subheader">Send to device</string>
+    <!-- An option from the share dialog to sign into sync -->
+    <string name="sync_sign_in">Sign in to Sync</string>
+    <!-- An option from the share dialog to send link to all other sync devices -->
+    <string name="sync_send_to_all">Send to all devices</string>
+    <!-- An option from the share dialog to reconnect to sync -->
+    <string name="sync_reconnect">Reconnect to Sync</string>
+    <!-- An option from the share dialog to reconnect to sync -->
```

Comment:
This is the same comment as above, and it seems like it should be updated.

Reply:
Update: Sawyer is updating this right now, so we should get that fix in with the next quarantine